db/re/item_db.txt

```diff
@@ -11178,6 +11226,7 @@
 21016,Two_Handed_Sword_of_Vicious_Mind,Two-Handed Sword of Vicious Mind,5,20,,2200,220,,1,1,0x00004082,63,2,34,4,160,1,3,{ bonus bAtk,pow(min(getrefine(),15),2); },{},{}
 21018,Lindy_Hop,Lindy Hop,5,20,,3400,340,,1,2,0x00004082,56,2,34,4,170,1,3,{ .@r = getrefine(); bonus2 bAddClass,Class_All,.@r/2; bonus bAspdRate,.@r; bonus bUnbreakableWeapon; },{},{}
 21019,Onimaru,Onimaru,5,0,,4200,75,,1,2,0x00000080,56,2,34,4,130,1,3,{ .@bStr = readparam(bStr); .@r = getrefine(); bonus bBaseAtk,(min(120,.@bStr)); if (.@bStr > 119) bonus bBaseAtk,160; else if (.@bStr > 107) bonus bBaseAtk,80; else if (.@bStr > 94) bonus bBaseAtk,40; if (.@r > 6) bonus bUnbreakableWeapon; bonus4 bAutoSpell,"NPC_WIDECURSE",4,100,0; if (.@r > 8) bonus4 bAutoSpellOnSkill,"LK_BERSERK","BS_OVERTHRUST",5,100; },{},{}
+21038,Oriental_Sword,Oriental Sword,5,,,1000,0,0,,2,0xFFFFFFFF,63,2,2,4,,,2,{ .@r = getrefine(); if (.@r >= 2) { bonus bBaseAtk,.@r/2; } if (.@r >= 3) { bonus2 bSkillAtk,"RK_SONICWAVE",.@r/3*10; } if (.@r >= 9) { bonus5 bAutoSpell,"RK_SONICWAVE",2,.@r/3*10,BF_SHORT|BF_WEAPON,1; } if (.@r >= 11) { bonus bUnbreakableWeapon; } },{},{}
```

Comment:
This item is a Two-handed sword 

So it should change location from 2 -> 34
https://www.divine-pride.net/database/item/21038